setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='sparkpost',
-    version='1.0.0.dev2',
+    version='1.0.0.dev3',
```

Comment:
let's not worry about changing the version yet - we're going to get this and scheduled send in and call it 1.0.0